packages/laconia-batch/src/laconiaBatch.js

```diff
@@ -9,6 +9,10 @@ const forwardEvents = (from, eventNames, to, laconiaContext) => {
   );
 };
 
+const isBatchProcessingNotStarted = cursor => {
+  return cursor === undefined;
```

Comment:
Should we check for explicitly `undefined`, or is a falsy check good enough, I.e. 

```
const isBatchProcessingNotStarted = cursor => !cursor
```

And if so, do we need the extra function itself?

Reply:
Cursor being an object it makes sense to do a falsy check. 

Having it as a function makes it descriptive and easy to understand.

Reply:
Good point. It's either a function or a comment, and you might be able to reuse the function, or it might get more complex down the road.